### PR TITLE
Drop packages that we spun out

### PR DESCRIPTION
**Pull request by almarklein**
* [x] Drop `flexx.pyscript` and use `pscript` module instead
* [x] Drop `flexx.webruntime` and use `webruntime` module instead
* [x] Drop `flexx.dialite` (webruntime uses it, Flexx itself not)
* [x] Add CI for Python 3.7! 🎆

### CODE REVIEW

**Comment by almarklein** [comment]
These are the best kinds of PR's :D

File: README.md [hunk 1]
```diff
@@ -64,8 +61,14 @@ to use.
 Installation
 ------------
 
-Flexx requires Python 3.5+ (only PyScript can be used with 2.7+) and also works on pypy. Further,
-it needs the [tornado](http://www.tornadoweb.org) library (pure Python).
+Flexx requires Python 3.5+ and also works on pypy. Further,
+it depends on:
+
+* the [tornado](http://www.tornadoweb.org) library (pure Python).
+* the [PScript](http://github.com/flexxui/webruntime) library (a flexxui project).
```
Comment (ivoflipse) [inline README.md]: These urls are incorrect

Comment (almarklein) [reply README.md]: Nice catch! I take it that you meticulously reviewed this whole PR? :P

Comment (ivoflipse) [reply README.md]: Nope TLDR

**Comment by almarklein** [comment]
These are the most fulfilling PR's to merge ...

**Pull request merged**